addons/website_sale_slides/models/product_product.py

```diff
@@ -9,6 +9,15 @@ class Product(models.Model):
 
     channel_ids = fields.One2many('slide.channel', 'product_id', string='Courses')
 
+    def _is_add_to_cart_allowed(self):
+        # Allow course related products to the cart regardless of product's rules
```

Comment:
Add in method docstring instead of python comment.

And modify to "Allow **published** course related..."